src/vos/evt_priv.h

```diff
@@ -359,53 +364,60 @@ int evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
  *
  * Order is lower high offset, higher epoch, lower high offset
  */
-int evt_rect_cmp(const struct evt_rect *rt1, const struct evt_rect *rt2);
+int
+evt_rect_cmp(const struct evt_rect *rt1, const struct evt_rect *rt2);
 
 /** Convert a context to a daos_handle
  * \param[IN]	tcx	The evtree context
  *
  * Returns the converted handle
  */
-daos_handle_t evt_tcx2hdl(struct evt_context *tcx);
+daos_handle_t
+evt_tcx2hdl(struct evt_context *tcx);
 
 /** Convert a handle to an evtree context
  * \param[IN]	handle	The daos handle
  *
  * Returns the converted handle
  */
-struct evt_context *evt_hdl2tcx(daos_handle_t toh);
+struct evt_context *
+evt_hdl2tcx(daos_handle_t toh);
 
 /** Move the trace forward.
  * \param[IN]	tcx	The evtree context
  */
-bool evt_move_trace(struct evt_context *tcx);
+bool
+evt_move_trace(struct evt_context *tcx);
 
 /** Get a pointer to the rectangle corresponding to an index in a tree node
  * \param[IN]	tcx	The evtree context
  * \param[IN]	node	The tree node
  * \param[IN]	at	The index in the node entry
+ * \param[out]	rout	Returned rectangle
  *
  * Returns the rectangle at the index
  */
-struct evt_rect *evt_node_rect_at(struct evt_context *tcx,
-				  struct evt_node *node, unsigned int at);
+void
+evt_node_read_rect_at(struct evt_context *tcx, struct evt_node *node,
+		      unsigned int at, struct evt_rect *rout);
 
 /** Get a pointer to the rectangle corresponding to an index in a tree node
  * \param[IN]	tcx	The evtree context
  * \param[IN]	nd_off	The offset of the tree node
  * \param[IN]	at	The index in the node entry
+ * \param[out]	rout	Returned rectangle
  *
  * Returns the rectangle at the index
  */
-static inline struct evt_rect *evt_nd_off_rect_at(struct evt_context *tcx,
-						  umem_off_t nd_off,
-						  unsigned int at)
+static inline void
+evt_nd_off_read_rect_at(struct evt_context *tcx, umem_off_t nd_off,
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?